### ISSUE TITLE
Case Sensitivity of ARIA ID References

### ISSUE THREAD

**Issue by WilcoFiers** [auto-wcag/auto-wcag#2282]
## Accessibility support

Some user agents treat the value of `aria-*` attribute as case-sensitive (even when these are not IDs) while some treat them as case-insensitive.

## ACT Rules

- [ARIA required ID references exist - in6db8](https://www.w3.org/WAI/standards-guidelines/act/rules/in6db8/proposed/#accessibility-support)

### Test case 1

Case sensitivity of ARIA ID references

```html
TODO
```

#### Test instructions

1. TODO

## Test results

Please report test results in a comment below. Please use the following format:

```md
## Test case 1

- Date: ...
- Operating system: ..., version ...
- Browser: ..., version ...
- Assistive technology: ..., version ...
- Test result: Passed / Failed (if failed, how was it different from expected?)
```
